source/security-and-authentication/mongodb-aws-auth.txt

```diff
@@ -56,10 +63,10 @@ replace:
      - Value of your ``AWS_SECRET_KEY``.
 
    * - **hostname**
-     - Network address of your MongoDB server.
+     - Network address of your {+cluster+}.
 
    * - **port**
-     - Port number of your MongoDB server.
+     - Port number of your {+cluster+}.
```

Comment:
Suggestion:
I think these reference a single node in the connection string (line 40) and not a cluster, so it might be better to leave them as "MongoDB server" for consistency.